docs/dg/dev/backend-development/audit-logs/audit-logs.md

```diff
@@ -0,0 +1,265 @@
+
+## Example of audit log data
+
+This example of an audit log entry was recorded during a successful login attempt:
+
+
+<details>
+  <summary>Audit log example</summary>
+```json
+{
+    "@timestamp": "2024-07-16T09:45:12.310532+00:00",
+    "@version": 1,
+    "host": "b0bc3b05abf2",
+    "message": "Successful Login",
+    "type": "YVES",
+    "channel": "security",
+    "level": "INFO",
+    "monolog_level": 200,
+    "extra": {
+        "environment": {
+            "application": "YVES",
+            "environment": "docker.dev",
+            "store": null,
+            "codeBucket": "EU",
+            "locale": "en_US"
+        },
+        "server": {
+            "url": "http://yves.de.spryker.local/en/login_check",
+            "is_https": false,
+            "hostname": "yves.de.spryker.local",
+            "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36",
+            "user_ip": "192.168.000.00",
+            "request_method": "POST",
+            "referer": "http://yves.de.spryker.local/en/login"
+        },
+        "request": {
+            "requestId": "502835fe",
+            "type": "WEB",
+            "request_params": {
+                "loginForm": {
+                    "email": "sonia@spryker.com",
+                    "password": "***",
+                    "_token": "f4dd629da0074c0b.b3QpFehJ8cMYAlDmDTcqm-se4sxEdz_BvLQXXmzo2iw.Lj1mRYQRifJTYDW2emhI0qNflZkMNQ-U6_xiLCbYjH8NOWAnrC-_hXF3aA"
+                },
+                "username": "sonia@spryker.com",
+                "customer_reference": "DE--21"
+            },
+            "log_type": "audit_log"
+        },
+        "context": {
+            "tags": ["successful_login"]
+        }
+    }
+}
+```
+
+</details>
+
+The data can be further enriched by creating and integrating custom processors. These processors can be registered within the plugins implementing `AuditLoggerConfigPluginInterface`. This extensibility ensures that audit logs can be tailored to meet specific requirements and provide deeper insights into application activities.
+
+For instructions, see [HowTo: Extend the log structure with additional data](/docs/pbc/all/miscellaneous/{{page.version}}/tutorials-and-howtos/how-to-extend-the-log-structure-with-additional-data.html).
+
+## Disallowing logging for specific tags
+
+If you don't want some actions to be logged, you can disallow logging for specific tags. In the following example, the `user_logged_in` tag is disallowed, preventing logs with this tag from being recorded.
+
+**config/Shared/config_default.php**
+
+```php
+<?php
+
+use Spryker\Shared\Log\LogConstants;
+
+$config[LogConstants::AUDIT_LOG_TAG_DISALLOW_LIST] = [
+    'user_logged_in',
+];
+
+```
+
+
+## Configuring the log path
+
+You can configure the log path to either a file or an output stream like `php://stdout`. If an application
```

Comment:
We have to provide more explanations
- when to use file and how to make a real use of it
- can we use ftp/ssh file?
- why we cannot use Flysystem here?

Reply:
If I'm not mistaken, local files in the container are not recommended, since they are removed with container restart, and customers don't have access to them. 
@dunqan what's your opinion here?

Reply:
fixed